doc/source/packaging/code/authors_code.rst

```diff
@@ -7,8 +7,8 @@
     # For contributions made under a Corporate CLA, the organization is
     # added to this file.
     #
-    # If you have contributed to the repository and wish to be added to this file
-    # please submit a request.
+    # If you have contributed to the repository and want to be added to this file,
+    # submit a request.
```

Comment:
Maybe we should clarify how we can submit a request.

Reply:
That's also true. I would expect the user to let it know in the PR. But we should clarify it